docs/ref/models/expressions.txt

```diff
@@ -515,15 +522,10 @@ The ``value`` argument describes the value to be included in the expression,
 such as ``1``, ``True``, or ``None``. Django knows how to convert these Python
 values into their corresponding database type.
 
-The ``output_field`` argument should be a model field instance, like
-``IntegerField()`` or ``BooleanField()``, into which Django will load the value
-after it's retrieved from the database. Usually no arguments are needed when
-instantiating the model field as any arguments relating to data validation
-(``max_length``, ``max_digits``, etc.) will not be enforced on the expression's
-output value. If no ``output_field`` is specified it will be tentatively
-inferred from the :py:class:`type` of the provided ``value``, if possible. For
-example, passing an instance of :py:class:`datetime.datetime` as ``value``
-would default ``output_field`` to :class:`~django.db.models.DateTimeField`.
+If no ``output_field`` is specified it will be tentatively inferred from the
+:py:class:`type` of the provided ``value``, if possible. For example, passing
+an instance of :py:class:`datetime.datetime` as ``value`` would default
+``output_field`` to :class:`~django.db.models.DateTimeField`.
```

Comment:
```suggestion
If no ``output_field`` is specified, it will be inferred from the type of the
provided ``value`` for many common types. For example, passing an instance of
:py:class:`datetime.datetime` as ``value`` defaults ``output_field`` to
:class:`~django.db.models.DateTimeField`.
```

Reply:
Applied!